images/virtio-container-disk/Dockerfile

```diff
@@ -16,19 +16,13 @@
 # Copyright 2017 Red Hat, Inc.
 #
 
-FROM fedora:28 as builder
+FROM kubevirt/container-disk-v1alpha
+
+LABEL maintainer="The KubeVirt Project <kubevirt-dev@googlegroups.com>"
 
 RUN curl https://fedorapeople.org/groups/virt/virtio-win/virtio-win.repo -o /etc/yum.repos.d/virtio-win.repo \
-&& dnf install -y virtio-win && mkdir -p /disk 
+&& dnf install -y virtio-win && dnf clean all && mkdir -p /disk
```

Comment:
Can you either use a symlink instead of cp, or "mv" the image in the same "RUN" statement? Otherwise we have the disk two times incluided.

Reply:
sure, done

Reply:
ah wait, according to the comment a symlink doesn't work...? Will change to mv

Reply:
mv doesn't follow symlinks, so I copy and delete now